src/destination/selector.js

```diff
@@ -108,6 +112,10 @@ export default function destinationSelectorFactory($container, config) {
                 }
             };
 
+            var getSelectedACLTransferMode = function getSelectedACLTransferMode() {
+                return $('input[name="acl-mode"]:checked').val();
+            };
```

Comment:
**thought:** New code can be written in ES6 in this repo. Actually, when possible, we can also convert existing code to ES6. Anyway not a blocker.

**suggestion:**
```suggestion
            const getSelectedACLTransferMode = () => $('input[name="acl-mode"]:checked').val();
```

Reply:
updated, thanks!